web/app/themes/mitlib-child/style.css

```diff
@@ -1,7 +1,7 @@
 /*
 Theme Name: MITlib Child
 Author: MIT Libraries
-Version: 1.0.2
+Version: 1.0.3
```

Comment:
I see that you're changing the version string for both themes, which is the right thing to do - no changes needed.

That said, I also wanted to mention that we are free to follow any convention we want with these version strings. There's nothing magical about semantic versioning here, and you saw that we don't use the same convention with the parent theme. Ideally we'd use the same convention for all internal version strings in this repo?

If you have thoughts, I'm happy to hear them. We can also talk about what practices we follow for other platforms if that would be helpful.

Reply:
Good point! Unless we're going to be doing a lot of updates I'm not sure if we need three layers of version numbers. If we're not worried about leaving room to increment the first number when there's a large update (is that the key component of semantic versioning?) then either path makes sense to me.

It seems like being consistent would be best regardless of which direction we go. What do we do outside of Wordpress?

Reply:
The convention we're following with the child theme - but not in a super strict sense, I don't think - is [semantic versioning](https://semver.org/). That makes the most sense to me for software packages which have an external audience - which doesn't apply to anything we're really doing with WordPress.

Outside of WordPress, we've been trending toward simplicity lately. We've done X.Y versioning for things like [ETD](https://github.com/MITLibraries/thing/releases), which did go through an extensive rewrite between versions 1 and 2 (in hindsight that probably should have just been a new application and codebase, though). Our other big Rails apps, [Bento](https://github.com/MITLibraries/bento/releases) and the [TIMDEX API](https://github.com/MITLibraries/timdex/releases), all have an X.Y format but we've specifically chosen to avoid touching the X digit at several points in the past. Same with [TACOS](https://github.com/MITLibraries/tacos/releases) - although we haven't ruled out going from v0.x to v1.x at some point when we've got all the bits we intend to include in place.

Reply:
Make sense... For the case of X.Y, if you get to X.9 and there's another update... would you just start adding digits? That would be my only concern is that you run out of runway if you're intentionally trying to avoid incrementing X.

Reply:
In that scenario, you go from x.9 to x.10 - the whole sequence is still a string, with only the individual digits treated numerically.

Reply:
Cool, I think that route seems simplest and I'm all for keeping things simple. I can collapse the child theme version to 1.3 with my next round of changes.